### ISSUE TITLE
Server - Kirin Tor   //  Quest - The Drakkensryd Quest ID 12886 (All ID numbers included in)

### ISSUE THREAD

**Issue by BlanketBearer** [dalaranwow/dalaran-wow#4791]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

On the Quest The Drakkensryd in The Storm Peaks. The quest does not mount you on a dragon at the beginning. The NPCs are difficult to kill, especially as melee because the game thinks that you aren't facing the correct direction. If you manage to kill an NPC, it does not count towards your completion of the quest.

**Expected behaviour**: Tell us what should happen instead

After talking to the NPC (Gretta the Arbiter) and accepting the quest The Drakkensryd, you should be automatically mounted onto a dragon. The dragon should fly you to the top of "The Temple of Storms." You can use a spear to move from dragon to dragon and kill the NPCs called Hyldsmeet Drakeriders. Each Hyldsmeet Drakerider killed should count as a kill, and you need to kill 10 of them to complete the quest.

**Steps to reproduce the problem**:

1. Accept the quest "The Drakkensryd" in the Storm Peaks from the NPC (Gretta the Arbiter) with a warrior or any melee character. Notice how the quest does not automatically mount you onto a dragon.
2. Fly yourself to the top of "The Temple of Storms" in the Storm Peaks. Approach a dragon with the target NPCs called (Hyldsmeet Drakerider). Use the gadget icon to board a dragon with the target NPC on it. 
3. Try to attack the target NPC. You will likely not be able to attack it, because the game doesn't think that it is in front of you. If you manage to kill one, notice that the kill does not count toward the quest objective.

**Include proofs for this behaviour**

http://wowwiki.wikia.com/wiki/Quest:The_Drakkensryd
https://www.youtube.com/watch?v=oCcyIi7sqZE

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

The Drakkensryd - Quest ID 12886
Gretta the Arbiter - NPC, ID 29796
Hyldsmeet Drakerider NPC, ID 29694

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Your issue posted here is a duplicate of issue: https://github.com/dalaranwow/dalaran-wow/issues/4349 (< click on this and you will be redirected to the original posted issue). 

As you can see the issue has been fixed by a developer and will be tested on the Public Test Realm in the future (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm in the future.
This means that the issue exists in game at the moment i wrote this message.
**Only once the issue was marked with "Fixed - On Live Server", you can find the fix on your game server.**
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.
